code/game/objects/items/weapons/abridged_tome.dm

```diff
@@ -0,0 +1,187 @@
+//Ordered by traitor librarians. Gives access to some weaker runes and provides basic tome functionality.
+/obj/item/weapon/abridged_tome
+	name = "arcane tome"
+	desc = "An old, dusty tome with dog-eared edges and a sinister-looking cover. Its pages look well-thumbed."
+	icon_state ="tome_old"
+	throw_speed = 2
+	throw_range = 5
+	w_class = WEIGHT_CLASS_SMALL
+	var/datum/mind/reader //The person "attuned" to this tome
+
+/obj/item/weapon/abridged_tome/proc/is_reader(mob/living/user) //Checks if the mob using the tome is the person attuned to it
+	return user && user.mind && user.mind == reader
+
+/obj/item/weapon/abridged_tome/examine(mob/living/user)
+	..()
+	if(is_reader(user) || user.stat == DEAD)
+		user << "<span class='cult'>An old version of the arcane tome no longer in circulation. This one reached #1 on Acribus' Top Reads list.</span>"
+		user << "<span class='cult'>Used to scribe certain runes, and can also be used as a powerful burn weapon.</span>"
+
+/obj/item/weapon/abridged_tome/attack(mob/living/M, mob/living/user)
+	if(!is_reader(user))
+		return ..()
+	if(iscultist(M))
+		if(M.reagents && M.reagents.has_reagent("holywater"))
+			user << "<span class='cult'>You remove the holy water from [M].</span>"
+			var/holy2unholy = M.reagents.get_reagent_amount("holywater")
+			M.reagents.del_reagent("holywater")
+			M.reagents.add_reagent("unholywater",holy2unholy)
+			add_logs(user, M, "smacked", src, " removing the holy water from them")
+		return
+	M.take_bodypart_damage(0, rand(5, 20)) //Damage done is a throwback to the damage that actual tomes used to do
+	playsound(M, 'sound/weapons/sear.ogg', 50, 1)
+	M.visible_message("<span class='danger'>[user] strikes [M] with the arcane tome!</span>", \
+					  "<span class='userdanger'>[user] strikes you with the tome, searing your flesh!</span>")
+	user.do_attack_animation(M)
+	add_logs(user, M, "smacked", src)
+
+/obj/item/weapon/abridged_tome/attack_self(mob/living/user)
+	if(!is_reader(user))
+		if(!reader)
+			user << "<span class='notice'>You cut open a finger and press a droplet of blood onto [src]'s pages. You can read them now.</span>"
```

Comment:
>plasma man librarian

i know some people like to shoot themselves in the foot but holy hell dude

in all seriousness I don't know what plasmamen cultists do about the blood situation but you could just copy that.